embulk-output-bigquery.gemspec

```diff
@@ -14,18 +14,13 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec)/})
   spec.require_paths = ["lib"]
 
-  # TODO
-  # signet 0.12.0 and google-api-client 0.33.0 require >= Ruby 2.4.
-  # Embulk 0.9 use JRuby 9.1.X.Y and it's compatible with Ruby 2.3.
-  # So, force install signet < 0.12 and google-api-client < 0.33.0
-  # Also, representable version >= 3.1.0 requires Ruby version >= 2.4
-  spec.add_dependency 'signet', '~> 0.7', '< 0.12.0'
-  spec.add_dependency 'google-api-client','< 0.33.0'
+  # the latest version
+  spec.add_dependency 'google-api-client','= 0.53.0'
   spec.add_dependency 'time_with_zone'
-  spec.add_dependency "representable", ['~> 3.0.0', '< 3.1']
-  # faraday 1.1.0 require >= Ruby 2.4.
-  # googleauth 0.9.0 requires faraday ~> 0.12
-  spec.add_dependency "faraday", '~> 0.12'
+  spec.add_dependency 'thwait'
+  # activesupport require Ruby >= 2.7.0
+  # jruby-9.3.0.0 is MRI 2.6 compatible
+  spec.add_dependency 'activesupport', "< 7.0"
```

Comment:
The single gem for bigquery API doesn't depend on activesupport.
This dependency is actually unnecessary.
But, I don't see a problem, because this PR is a first porting.